app/assets/javascripts/initializers/initializeDrawerSliders.js

```diff
@@ -1,57 +1,66 @@
+'use strict';
+
+function listenForNarrowMenuClick() {
+  var navLinks = document.getElementsByClassName('narrow-nav-menu');
+  var narrowFeedButt = document.getElementById('narrow-feed-butt');
+  var narrowNavMenu = document.getElementById('narrow-nav-menu');
```

Comment:
something is odd here, the `navLinks` are an array of items with class `.narrow-nav-menu` and the `narrowNavMenu` is an item whose ID is `#narrow-nav-menu`. If I grep the code for `narrow-nav-menu` there's only one file containing that string - https://github.com/thepracticaldev/dev.to/blob/master/app/views/stories/_narrow_nav_menu.html.erb - which makes me think there's something odd going on. Why are we iterating on a list of one to remove a class from the same exact element? And then we add event handlers to add and remove the `showing` class.

I'm not 100% sure about what this function is for but it looks like me the code can be simplified.

Pinging @benhalpern who might know more about this